Directory.Build.props

```diff
@@ -4,8 +4,9 @@
     <RepoEngPath>$(MSBuildThisFileDirectory)eng</RepoEngPath>
     <RepoSrcPath>$(RepoRoot)src</RepoSrcPath>
     <RepoSdkPath>$(RepoRoot)src/SDKs</RepoSdkPath>
-    <IsClientLibrary Condition="'$(IsClientLibrary)' == '' and $(MSBuildProjectName.StartsWith('Azure.'))">true</IsClientLibrary>
-    <IsDataPlaneProject Condition="'$(IsDataPlaneProject)' == '' and '$(IsClientLibrary)' == 'true'">true</IsDataPlaneProject>
+    <IsLegacyMgmtProject Condition="'$(IsLegacyMgmtProject)' == '' and $(MSBuildProjectName.Contains('.Management.'))">true</IsLegacyMgmtProject>
+    <IsMicrosoftProject Condition="$(MSBuildProjectName.StartsWith('Azure.')) or $(MSBuildProjectName.StartsWith('Microsoft.'))">true</IsMicrosoftProject>
```

Comment:
I don't think the name correlates well with the condition.

Reply:
Yeah I agree it isn't the best. I essentially used this to filter out our common imports for anything that didn't look like a shipping project. Mostly there were samples so I guess we could invert the logic.